pkg/volume/vsphere_volume/attacher.go

```diff
@@ -86,34 +86,57 @@ func (attacher *vsphereVMDKAttacher) Attach(spec *volume.Spec, nodeName types.No
 }
 
 func (attacher *vsphereVMDKAttacher) VolumesAreAttached(specs []*volume.Spec, nodeName types.NodeName) (map[*volume.Spec]bool, error) {
-	volumesAttachedCheck := make(map[*volume.Spec]bool)
+	glog.Warningf("Attacher.VolumesAreAttached called for node %q - Please use BulkVerifyVolumes for vSphere", nodeName)
```

Comment:
Can we change warningf to infof?

Reply:
No, we want to use BulkVerifyVolumes() API instead of VolumesAreAttached() API. So any invocations on VolumesAreAttached() should be a warning log.